build.gradle.kts

```
@@ -7,7 +7,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 plugins {
-  id("org.jetbrains.kotlinx.kover") version "0.7.3" apply false
+  id("org.jetbrains.kotlinx.kover") version "0.7.4" apply false
```

```suggestion
  id("org.jetbrains.kotlinx.kover") version "0.7.3" apply false
```

https://github.com/Kotlin/kotlinx-kover/issues/478